src/components/web/WorkBoard.jsx

```
@@ -7,47 +7,69 @@ import popular from '../../assets/icons/common/common_tag_popular.svg'
+
+const getEmploymentLabel = (value) => {
+  const found = employmentTypes.find((item) => item.value === value)
+  return found ? found.label : value
+}
 
 const WorkBoard = ({
   title,
   name,
   date,
-  like: initialLike,
+  like,
```

ESLint

[error] 'like' is defined but never used.

(no-unused-vars)